packages/flutter/lib/src/cupertino/text_field.dart

```diff
@@ -796,10 +796,6 @@ class CupertinoTextField extends StatefulWidget {
   /// By default, builds a [CupertinoTextMagnifier] on iOS and Android nothing on all other
   /// platforms. If it is desired to supress the magnifier, consider passing
   /// [TextMagnifierConfiguration.disabled].
-  ///
-  // TODO(antholeole): https://github.com/flutter/flutter/issues/108041
-  // once the magnifier PR lands, I should enrich this area of the
-  // docs with images of what a magnifier is.
```

Comment:
Do you want to include these examples here or anywhere else in your docs?  You can do it like this:

https://github.com/flutter/flutter/blob/965912d88631691dbf14457ef481e1626de4520f/packages/flutter/lib/src/widgets/text.dart#L355-L360